demos/python_demos/clip_image_classification/download_model_requirements.txt

```diff
@@ -1,3 +1,5 @@
+--extra-index-url https://download.pytorch.org/whl/cpu
 openvino>=2023.2.0
 transformers==4.36.2
 pillow==10.1.0
```

Comment:
The demo stopped to work after this PR.
https://jira.devtools.intel.com/browse/CVS-130850

Please also add data and model dirs into gitignore